content/en/cloud_cost_management/custom_allocation_rules.md

```diff
@@ -9,74 +9,127 @@ further_reading:
+In the example above, EC2 support fees across teams are allocated based on their share of overall EC2 spend. Proportions are calculated based on how much each team spent.
+
+To create a rule for this allocation, you could:
+
+- Define the costs to allocate (source): **EC2 support fees** (`aws_product:support`).
+- Choose the allocation method: **Proportional by spend**.
+- Define the allocation destination: **Team** (`teamA`, `teamB`, `teamC`).
+- Define the proportions to split the source costs : **EC2** (`aws_product:ec2`).
+
+You can also specify how cost proportions should be partitioned to ensure segment-specific allocations. For example, if you partition your costs by environment using tags like `staging` and `production`, the proportions are calculated separately for each environment. This ensures allocations are based on the specific proportions within each partition.
 
-{{< img src="cloud_cost/custom_allocation_rules/proportional_ui.png" alt="The proportional split strategy as seen in Datadog" style="width:60%;" >}}
+{{< img src="cloud_cost/custom_allocation_rules/ui-proportional-by-spend.png" alt="The proportional split strategy as seen in Datadog" style="width:90%;" >}}
 
 {{% /tab %}}
 
-{{% tab "Dynamic by Metric Allocation" %}}
+{{% tab "Dynamic by metric" %}}
 
-Metrics-based allocation provides the ability to split up costs based on Datadog's [metrics queries][3]. By using performance metrics to allocate expenses, you can more accurately allocate costs based on application usage patterns.
+{{< img src="cloud_cost/custom_allocation_rules/dynamic_diagram.png" alt="Diagram illustrating the dynamic by metric strategy" style="width:70%;" >}}
 
-For example, this PostgreSQL metrics query `sum:postgresql.queries.time{*} by {user}.as_count()` tracks the total query execution time per user. The relative values are then used to determine what proportion of total PostgreSQL costs should be allocated to each user.
+Metrics-based allocation provides the ability to split up costs based on Datadog's [metrics queries][1]. By using performance metrics to allocate expenses, you can more accurately allocate costs based on application usage patterns.
 
-For determining the proportion of costs to be allocate, metrics can be aggregated on a daily or monthly basis. However, the costs themselves are still allocated on a daily basis.
+For example, this PostgreSQL metrics query `sum:postgresql.queries.time{*} by {user}.as_count()` tracks the total query execution time per user. The relative values are then used to determine what proportion of total PostgreSQL costs should be allocated to each user, as follows:
 
-{{< img src="cloud_cost/custom_allocation_rules/dynamic_diagram.png" alt="Diagram illustrating the dynamic by metric strategy" style="width:60%;" >}}
+To create a rule for the above allocation, you could:
```

Comment:
I'm not sure if you mean the diagram or the example in the paragraph directly before this sentence. If it's the former, I recommend stating the "preceding diagram" and if it's the latter, I recommend moving the sentence into that paragraph.

```suggestion
To create a rule for this allocation, you could:
```